R/weather_radars.R

```diff
@@ -1,6 +1,6 @@
-#' Get weather radar data
+#' Get weather radar metadata
 #'
-#' The `weather_radars()` function is used to retreive weather radar metadata
+#' The `weather_radars()` function is used to retrieve weather radar metadata for European weather radars
```

Comment:
Now thinking what if the US gets added how would we add functionality to the package to retrieve US radars. As an argument to this function? or as an separate function? @CeciliaNilsson709 do you have preferences or an idea?

Reply:
Conclusion one function will cover both continents

Reply:
Combining columns we will figure out later